internal/namespaces/instance/v1/custom_server_test.go

```diff
@@ -279,6 +279,49 @@ func Test_ServerDelete(t *testing.T) {
 	interactive.IsInteractive = false
 }
 
+// These tests needs to be run in sequence
+// since they are using the interactive print
+func Test_ServerTerminate(t *testing.T) {
+	interactive.IsInteractive = true
+
+	t.Run("without IPs", core.Test(&core.TestConfig{
+		Commands:   GetCommands(),
+		BeforeFunc: core.ExecStoreBeforeCmd("Server", "scw instance server create image=ubuntu-bionic -w"),
+		Cmd:        `scw instance server terminate {{ .Server.ID }}`,
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(0),
+		),
+		AfterFunc:       core.ExecAfterCmd(`scw instance ip delete {{ index .Server.PublicIP.ID }}`),
+		DisableParallel: true,
+	}))
+
+	t.Run("with IPs", core.Test(&core.TestConfig{
+		Commands:   GetCommands(),
+		BeforeFunc: core.ExecStoreBeforeCmd("Server", "scw instance server create image=ubuntu-bionic -w"),
+		Cmd:        `scw instance server terminate {{ .Server.ID }} with-ip=true`,
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(0),
+		),
+		DisableParallel: true,
+	}))
+
+	t.Run("without block", core.Test(&core.TestConfig{
+		Commands:   GetCommands(),
+		BeforeFunc: core.ExecStoreBeforeCmd("Server", "scw instance server create image=ubuntu-bionic additional-volumes.0=block:10G -w"),
+		Cmd:        `scw instance server terminate {{ .Server.ID }} with-ip=true with-block=false`,
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(0),
```

Comment:
We should add CheckFunc that test the volume still exists

Reply:
I added another test with `with-block=true`, then added a `Check` to ensure that the volume is deleted.

Problem is: the volume deletion is not immediate (after an instance is terminated), and so the check fails because the volume is still there.

Reply:
cf. https://github.com/scaleway/scaleway-cli/issues/1000